src/bin/composer-requirements.py

```diff
@@ -11,13 +10,12 @@ def merge_requirements(env_data, local_data):
     env_require = env_data['require']
 
     for package in env_require:
-        if package in local_data['require'].keys():
-            local_data['require'][package]=env_require[package]
-            print('Found {0}: Replacing with {1}'.format(package, env_require[package]))
+        local_data['require'].update(env_require)
```

Comment:
I think that makes the `for` loop redundant , you can remove it (well this whole function can become one line, but we could keep it in case we need a more complex merge policy)

Reply:
Updated 👍🏻